challenges/challenge-weather-app/app.js

```diff
@@ -0,0 +1,82 @@
+// OpenWeather API key and endpoint
+const openWeatherAPIKey = config.openWeatherAPIKey;
+const openWeatherEndpoint = "http://api.openweathermap.org/data/2.5/weather";
+
+// Unsplash API key and endpoint
+const unsplashAccessKey = config.unsplashAccessKey;
+const unsplashEndpoint = "https://api.unsplash.com/search/photos";
+
+// Default city
+const defaultCity = "London";
+
+// Function to get weather data for a city
+async function getWeatherData(city) {
+  const url = `${openWeatherEndpoint}?q=${city}&appid=${openWeatherAPIKey}&units=metric`; // Celsius units
+  const response = await fetch(url);
+  const data = await response.json();
+
+  // Extracting weather description (this will be used to search for images)
+  const description = data.weather[0].description;
+
+  // Displaying the weather in the UI
+  document.getElementById(
+    "conditions"
+  ).textContent = `Weather in ${city}: ${description}`;
+
+  // Return the weather description for the Unsplash search
```

Comment:
I don't think Unsplash is related to this method. Please, consider updating the comment or removing it.